components/raftstore/src/store/config.rs

```diff
@@ -220,12 +217,11 @@ impl Default for Config {
             raft_max_size_per_msg: ReadableSize::mb(1),
             raft_max_inflight_msgs: 256,
             raft_entry_max_size: ReadableSize::mb(8),
-            raft_log_gc_tick_interval: ReadableDuration::secs(10),
-            raft_log_gc_threshold: 50,
+            raft_log_gc_tick_interval: ReadableDuration::secs(1),
```

Comment:
Please remember to update docs. https://github.com/pingcap/docs-cn/blob/master/tikv-configuration-file.md#raft-log-gc-tick-interval